lib/globus/client/endpoint.rb

```diff
@@ -84,8 +69,8 @@ def connection
         )
       end
 
-      def identity
-        Globus::Client::Identity.new(config)
+      def user
+        Identity.new(config).get_identity_id(user_id)
       end
```

Comment:
This class doesn't really care about identities; it just needs a user to fill out the `principal` field within an access request.